llvm/lib/CodeGen/SelectionDAG/TargetLowering.cpp

```diff
@@ -3184,6 +3191,20 @@ bool TargetLowering::SimplifyDemandedVectorElts(
     }
     break;
   }
+  case ISD::FREEZE: {
+    SDValue N0 = Op.getOperand(0);
+    if (TLO.DAG.isGuaranteedNotToBeUndefOrPoison(N0, DemandedElts,
+                                                 /*PoisonOnly*/ false))
```

Comment:
```suggestion
                                                 /*PoisonOnly=*/ false))
```

Reply:
Sorry, still not used to that code style :)